sdks/python/apache_beam/runners/interactive/caching/streaming_cache.py

```diff
@@ -63,6 +63,14 @@ def path(self):
     """Returns the path the sink leads to."""
     return self._path
 
+  @property
+  def size_in_bytes(self):
+    """Returns the space usage in bytes of the sink."""
+    try:
+      return os.stat(self._path).st_size
```

Comment:
Does this work in all OSes?

Reply:
It works. Though in some versions of some OSes, some of the fields of the returned stat might be dummy values.